examples/controlpanel/lib/uploadToMinio.js

```diff
@@ -0,0 +1,12 @@
+const uploadToMinio = async (file, url) => {
```

Comment:
@pozylon this component is there for demonstration purposes for the signed put URL upload and will be removed completely no need to change front end since buffer is supported